### PR TITLE
Lf 2454 : Sensor readings sync fix with ensemble request body

### PR DESCRIPTION
**Pull request by anant00729**
To test:

1. Unclaim sensors using bulk.py. [please connect with me]
2. Add sensors using a CSV file.
3. check if the webhook is able to send sensor readings local database.

### CODE REVIEW

**Review by wasswarichard — APPROVED** [review]

**Comment by wasswarichard** [comment]
@anant00729 it looks fine for me

**Comment by anant00729** [comment]
@Duncan-Brain could you please review.

File: packages/api/src/controllers/sensorController.js [hunk 1]
```diff
@@ -400,40 +400,57 @@ const sensorController = {
...
+            const row = {
+              read_time: sensorInfo.timestamps[k] || '',
+              location_id: corresponding_sensor.location_id,
+              value: sensorInfo.values[k],
+              reading_type: parameter_number,
+              valid: sensorInfo.validated[k] || false,
+              unit,
+            };
+            // Only include this entry if all required values are populated
+            if (Object.values(row).length) {
```
Comment (Duncan-Brain) [inline packages/api/src/controllers/sensorController.js:436]: @anant00729 I still don't think this replacement works. Does the comment still describe what it is trying to do?

**Comment by Duncan-Brain** [comment]
@anant00729 What is the main reason you think behind the previous failure? Is it mainly the return statements you think were preventing the responses?

**Review by Duncan-Brain — APPROVED** [review]
So it sounds like the return statements, and req.body check at the start, are the main fixes? Only one way to see if it works on beta!

**Pull request merged**